pdx-extract/app/sanitize.py

```diff
@@ -0,0 +1,89 @@
+import html
+
+
+def get_courses(subjects_json):
+    courses = []
+    for discipline in subjects_json:
+        for course in discipline:
+            courses.append(get_course_data(course))
+    return courses
+
+
+def get_course_data(course):
+    course_data = dict()
+
+    course_data["number"] = f"{course['subject']} {course['courseNumber']}"
+    course_data["name"] = html.unescape(course["courseTitle"])
+    course_data["crn"] = int(course["courseReferenceNumber"])
+    course_data["discipline"] = html.unescape(course["subjectDescription"])
+    course_data["days"] = get_days(course)
+    course_data["credits"] = int(course["creditHours"]) if course["creditHours"] else 0
+    course_data["time"] = get_time(course)
+    course_data["instructor"] = get_instructor(course)
+    course_data["term_description"] = get_term_description(course)
+    course_data["term_date"] = int(course["term"])
+
+    return course_data
+
+
+def get_time(record):
+    try:
+        meeting_time = record["meetingsFaculty"][0]["meetingTime"]
```

Comment:
This could just be that I'm not very used to Python, but `0` is arguably a magic number here. Consider using a constant variable with a meaningful name

Reply:
There is no specific meaning here, it's just how the data is structured. The list only has one element at index zero, always. Probably should document that better.